sematic/plugins/metrics_storage/sql/sql_metrics_storage.py

```diff
@@ -156,8 +158,10 @@ def get_aggregated_metrics(
         if isinstance(rollup, int):
             interval_seconds = max(interval_seconds, rollup)
             field_ = (
-                func.extract("epoch", MetricValue.metric_time)
-                / interval_seconds
+                sql_cast(
+                    func.extract("epoch", MetricValue.metric_time) / interval_seconds,
```

Comment:
In 2.0, this trick of `x / y * y` as a way to "round" to the nearest multiple of y stopped working as `x / y` gave a DB float type instead of an integer. This makes the integer cast explicit.